test/e2e/scheduler_predicates.go

```diff
@@ -110,6 +110,22 @@ func verifyResult(c *client.Client, podName string, ns string) {
 			"reason":                   "FailedScheduling",
 		}.AsSelector())
 	expectNoError(err)
+	// If we failed to find event with a capitalized first letter of reason
+	// try looking for one starting with a small one for backward compatibility.
+	// If we don't do it we end up in #158061.
+	// TODO: remove this block when we don't care about supporting v1.0 too much.
```

Comment:
I sent an email to the team about this.  We're trying to track issues that would have broken 1.0 but for hacks, so that when we have the conversation about dropping support for 1.0, we know what we're getting ourselves into.  @davidopp can weigh in.